package.json

```diff
@@ -4,17 +4,21 @@
   "description": "The main API for the texvet.io application.",
   "main": "",
   "scripts": {
-    "build": "sls package --stage development",
     "build:demo": "sls package --stage demo",
-    "build:prod": "sls package --stage production",
+    "build:development": "sls package --stage development",
+    "build:production": "sls package --stage production",
     "build:staging": "sls package --stage staging",
+    "compose": "docker-compose -f ./database/docker-compose.yml up -d ",
+    "seed:demo": "npm run compose && NODE_ENV=demo node ./dist/bundle.seeder.cjs.js",
+    "seed:development": "npm run compose && NODE_ENV=development node ./dist/seeder.cjs.js",
```

Comment:
These 3 commands are pointing at a non-existing script...should be pointing at ./dist/bundle.seeder.cjs.js